core-bundle/src/EventListener/DataContainer/ToggleNodesLabelListener.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\EventListener\DataContainer;
+
+use Contao\DataContainer;
+use Contao\DC_Table;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\HttpFoundation\Session\Attribute\AttributeBagInterface;
+
+class ToggleNodesLabelListener
+{
+    /**
+     * @var RequestStack
+     */
+    private $requestStack;
+
+    public function __construct(RequestStack $requestStack)
+    {
+        $this->requestStack = $requestStack;
+    }
+
+    public function __invoke(string $table): void
+    {
+        if (
+            DC_Table::class !== DataContainer::getDriverForTable($table)
```

Comment:
Why are you limiting this to `DC_Table`? There is a "toggle nodes" button in the file manager, too.

Reply:
added in 9226ea5ffbdc32bafb104d4607c73ff67f6fe3f3